File: cdd/LeetCode/LeetCode/convertToTitle.cpp
#include <stdio.h>
#include <string>

using namespace std;

class solution {
public:
	string convertToTitle(int columnNumber) 
	{
		string res;
		while (columnNumber >0)
		{
			--columnNumber;
			res += columnNumber % 26 + 'A';
			columnNumber /= 26;
		}
		reverse(res.begin(), res.end());
		return res;
	}

};

File: go/LeetCode/convertToTitle.go
package main

func main() {
	
}

func convertToTitle(columnNumber int) string {
	res := []byte{}

	for columnNumber>0 {
		columnNumber--
		res = append(res, 'A'+byte(columnNumber%26))
		columnNumber/=26
	}

	for i,n:=0,len(res); i<n/2;i++ {
		res[i],res[n-1-i] = res[n-1-i], res[i]
	}
	return string(res)
}
